src/core/qgsfeaturerequest.h

```diff
@@ -230,6 +230,18 @@ class CORE_EXPORT QgsFeatureRequest
          */
         CORE_EXPORT OrderBy( const QList<QgsFeatureRequest::OrderByClause> &other );
 
+        /**
+         * Equality operator
+         * \since QGIS 3.8
```

Comment:
```suggestion
         * \since QGIS 3.38
```

But strictly speaking, we don't require docs for equality /inequality operators, so you could just remove these.